src/vstart.sh

```diff
@@ -1162,6 +1162,22 @@ EOF
 	fi
         ceph_adm config-key set mgr/cephadm/ssh_identity_key -i ~/.ssh/id_rsa
         ceph_adm config-key set mgr/cephadm/ssh_identity_pub -i ~/.ssh/id_rsa.pub
+        local retry_flag=0
+        while [ $retry_flag -lt 7 ]
+        do
+            if [ `ceph_adm mgr stat | jq '.available'` = "true" ]; then
+                ceph_adm mgr module enable cephadm
+                break
```

Comment:
this should probably just `break`, if we're calling `ceph_adm mgr module enable cephadm` again just after the loop

Reply:
I delete `ceph_adm mgr module enable cephadm` after the loop but keep this in here, cuz when mgr is not available,  this should be run.

Reply:
IMO @cbodley is right, this should be just `break` and `ceph_adm mgr module enable cephadm` should stay where it was to keep the happy path ("mgr is available") and the unhappy path ("wait for mgr to become available and exit if that doesn't happen") separate.